docs/pages/admin-guides/access-controls/access-requests/automatic-reviews.mdx

```diff
@@ -0,0 +1,134 @@
+still allowing users to receive temporary access without manual approval.
+
+## How it works
+
+Automatic reviews are triggered by Access Monitoring Rules. These rules instruct
+Teleport to monitor specific Access Requests and automatically submit a review
+when certain conditions are met. The conditions can be configured based on the
+requested roles, and based on the requesting user's Teleport traits.
+
+Example:
+An Access Monitoring Rule can configure automatic Access Request approval when a
+user with the Teleport traits `team: demo` requests access to the `access` role.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+## Step 1/4. Create Requester Role and User
+
+In this example, we'll first create:
+- A role named `demo-access-request`, which allows requesting access to the
+`access` role.
+- A user named `demo-access-requester`, assigned the above role.
+
+### Role Definition
+
+Create a role configuration file named `demo-role.yaml`:
+```yaml
+# demo-role.yaml
+kind: role
+version: v7
+metadata:
+  name: demo-access-request
+spec:
+  allow:
+    request:
+      roles:
+      - access
+      search_as_roles:
+      - access
+```
+
+Create the role with:
+```sh
+$ tctl create demo-role.yaml
+```
+
+### Create the User
+
+Use the following command to create the user and assign the role:
+```sh
+$ tctl users add --roles=demo-access-request demo-access-requester
+```
+
+Alternatively, you can assign the role after creating the user:
+
+  (!docs/pages/includes/add-role-to-user.mdx role="demo-access-request" user="\`demo-access-requester\`"!)
+
+## Step 2/4. Assign User Traits
+
+To allow automatic review rules to evaluate the requesting user, assign them
+traits via the Teleport Web UI.
+
+1. Go to **Zero Trust Access** -> **Users**
+2. Next to `demo-access-requester`, click **Options** -> **Edit...**
+3. Click **Add user trait**, and set:
+   - Key: `team`
+   - Value: `demo`
+4. Click **Save**
+5. Verify that the user has been updated with the desired trait.
+
+![Edit User Traits](../../../../img/access-controls/automatic-reviews/users-edit.png)
+
+## Step 3/4. Create Access Monitoring Rule
+
+Next, define the automatic review rule via the Teleport Web UI.
+
+1. Go to **Identity Governance** -> **Access Requests** -> **View Access Monitoring Rules**
+2. Click **Create Access Monitoring Rule** -> **Automatic Review Rule**
+3. Configure the rule and set:
+   - **Name of roles to match**: `access`
+   - **User Traits**: `team: demo`
+   - **Review decision**: `APPROVED`
+4. Click **Create Rule**
+
+![Create Automatic Review Rule Editor](../../../../img/access-controls/automatic-reviews/create-automatic-review-rule.png)
+
+This Access Monitoring Rule will ensure that Access Requests for the `access`
+role will be automatically reviewed for approval if the Teleport user traits
+requirements are satisfied. In this case, any user with the traits `team: demo`
+will satisfy the requirement.
+
```

Comment:
Is the value of the **Key** field hardcoded, or does the user type it in? Does **Key** or **Value** support wildcards/regular expressions?

Reply:
The key field is not hardcoded. The user can specify any value. Wildcards/regular expressions are not supported for key or value in this format.